tests/testthat/test-workflow-dryrun.R

```diff
@@ -129,7 +34,7 @@ withr::with_options(list(rbabylon.model_directory = NULL), {
         test_yaml_path <- .test_case$test_yaml_path
 
         # load model from yaml
-        this_mod <- read_model(.path = test_yaml_path)
+        this_mod <- read_model(fs::path_ext_remove(test_yaml_path))
```

Comment:
this is kind of nit-picky but let's just clean up this pattern a little bit instead of using `fs::path_ext_remove` on the read_model calls. We don't actually _use_ the YAML path anywhere in this test, it was just for reading the model.

I propose the following:
- remove line 9 (where we define `TEST_YAML_DRY`)
- in lines 17-22
  - change `TEST_YAML_DRY` to `MOD_ID`
  - change `test_yaml_path` to `test_path`
- remove lines 34 and 87
- on lines 37 and 90 change `fs::path_ext_remove(test_yaml_path)` to `.test_case$test_path`

Better to clean this up now than to look at it later and wonder why we store yaml paths and then strip the extension off them.